models/public/colorization-v2-norebal/colorization-v2-norebal.md

```diff
@@ -0,0 +1,105 @@
+# colorization-v2-norebal
+
+## Use Case and High-Level Description
+
+The `colorization-v2-norebal` model is one of the [colorization](https://arxiv.org/pdf/1603.08511)
+group of models designed to perform image colorization. For details
+about this family of models, check out the [repository](https://github.com/richzhang/colorization).
+
+The gray-scale or BGR image should be convert to LAB image before inference.
```

Comment:
Because you deleted information about preprocessing, this sentence should be rephrased. Describe that model consumes as input (L-channel of LAB-image, AFAIK). And remove mention about conversion from ` gray-scale or BGR image`.

Also add description about model output.

*Here and in colorization-v2.md*

Reply:
@eizamaliev corrected